docs/man/git-lfs-fetch.1.ronn

```diff
@@ -25,10 +25,11 @@ This does not update the working copy.
   that would otherwise be downloaded. See [RECENT CHANGES]
 
 * `--all`:
-  Download all objects referenced by any commit that is reachable; this is
-  primarily for backup / migration purposes. Cannot be combined with --recent or
-  --include/--exclude. Ignores any globally configured include and exclude paths
-  to ensure that all objects are downloaded.
+  Download all objects that are referenced by any commit reachable from the refs
+  provided as arguments. If no refs are provided, then all refs are pushed. This
```

Comment:
I think we probably want to say "all refs are downloaded", right?

Reply:
Right. I don’t know why I didn’t spot this one :sweat_smile:. I’ll fix this right away.

Reply:
I used the wording “all refs are fetched” if that’s okay.